### PR TITLE
[Feature] Project "Status update"#6995

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6995

** UI & database changes, test in moped-test.
https://moped-test.austinmobility.io/moped/projects/161?tab=comments

https://user-images.githubusercontent.com/5282430/135158757-761fc95a-bec5-4af1-8df1-fe96f69aeba2.mp4

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectSummary.js [hunk 1]
```diff
@@ -28,7 +106,81 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
...
+                project_note_type: 2,
+              },
+            }
+          : {
+              project_note_id: { _eq: Number(commentId) },
+              added_by: addedBy,
+              project_note: statusUpdate,
+            }),
+      },
+    })
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectSummary.js:160]: This is the core of it, basically it checks whether this is an insert of a new comment, or an update of an existing one, and based on that condition it selects the function to execute and what variables to pass.

Kind of a convoluted summarized function, but I did not want it to be over-repetitive.

File: moped-editor/src/views/projects/projectView/ProjectSummary.js [hunk 2]
```diff
@@ -5,21 +5,99 @@ import ProjectSummaryTable from "./ProjectSummaryTable";
...
+    if (lastItem >= 0) {
+      // Get the data from the note
+      const note = data
+        ? data?.moped_project[0].moped_proj_notes[lastItem][fieldName] ?? ""
+        : null;
+      // Remove any HTML tags
+      return note ? String(note).replace(/(<([^>]+)>)/gi, "") : null;
+    }
+    return null;
+  };
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectSummary.js:92]: An important function, basically retrieves the last comment of type 2 and makes that the last status update.

**Comment by amenity** [comment]
Wow!! This works great, @sergiogcx! 😍

I had one UI refinement: 
Use the [multiline standard](https://mui.com/components/text-fields/#multiline) field type for "Current status" like we're using for "Description". It should behave the same way:
 - Default state: Label renders as h4, no horizontal rule, multiline rendering of text
 - Editing statue: horizontal rule, [field label scoots to the top](https://user-images.githubusercontent.com/1463708/135194737-4dd1b2d3-7bd3-4817-8714-66a20debc267.png), no grey background

I'll also assign @patrickm02L to this for extra design eyezzz

**Comment by sergiogcx** [comment]
@amenity It's done

**Comment by amenity** [comment]
One teeny thing - "Status Update" sentence case to "Status update," please! Otherwise looks 👌perfect!!

**Review by amenity — APPROVED** [review]
LSTM!✨🙏

**Review by chiaberry — APPROVED** [review]
+💯  for including the video demo!!

**Pull request merged**